index.html

```diff
@@ -16,12 +16,25 @@
 <body>
  <div class="container">
   <header>
+   <div class="hamburger-wrapper">
```

Comment:
Wydaje mi się że to pisałem w komentarzu - mniejsza z tym. Hamburger wrapper miał być w <nav> - tutaj docelowo stworzymy nawigację

Reply:
Poprawione